packages/app/src/sections/futures/Trade/TradeBalance.tsx

```diff
@@ -29,19 +30,50 @@ import CrossMarginInfoBox from '../TradeCrossMargin/CrossMarginInfoBox'
 
 import SmartMarginOnboardModal from './SmartMarginOnboardModal'
 
-type TradeBalanceProps = {
-	isMobile?: boolean
+type BrdigeAndWithdrawButtonProps = {
+	modalType: ModalType
+	onPillClick: () => void
+	expanded: boolean
 }
 
-const TradeBalance: React.FC<TradeBalanceProps> = memo(({ isMobile = false }) => {
+const BrdigeAndWithdrawButton: FC<BrdigeAndWithdrawButtonProps> = ({
+	modalType,
+	onPillClick,
+	expanded,
+}) => {
+	const dispatch = useAppDispatch()
+
+	return (
+		<FlexDivRowCentered columnGap="15px">
+			<PencilButton
+				width={16}
+				height={16}
+				onClick={(e) => {
+					e.stopPropagation()
+					dispatch(setOpenModal(modalType))
+				}}
+			/>
+			<Pill roundedCorner={false} onClick={onPillClick}>
+				<StyledCaretDownIcon $flip={expanded} style={{ marginTop: '1.5px' }} />
+			</Pill>
+		</FlexDivRowCentered>
+	)
+}
+
+const TradeBalance = memo(() => {
 	const { t } = useTranslation()
 	const dispatch = useAppDispatch()
+
 	const { deviceType } = useWindowSize()
+	const isMobile = deviceType === 'mobile'
```

Comment:
Could we wrap these in memo and move below selectors

Reply:
Update. Thanks.